roles/website/tasks/main.yaml

```diff
@@ -0,0 +1,96 @@
+---
+- name: Add user
+  user: name={{ website_user }} home=/home/{{ website_user }} shell=/bin/zsh
+
+- name: Create needed folder structure
+  file: path={{ item }} state=directory recurse=yes owner={{ website_user }} group={{ website_group }} mode=0755
```

Comment:
`0755` ist für die meisten dieser Verzeichnisse (eigentlich alle in `/home`) viel zu großzügig, `0700` wäre passender. Falls "Add user" oben bereits das Home mit `0700` anlegt ist das zu verkraften, das bezweifle ich aber (und [die Doku](https://docs.ansible.com/ansible/user_module.html) schweigt sich dazu aus).